### PR TITLE
TELCODOCS-1562 - RNs - New Features - Telco RAN 4.15

### PR DESCRIPTION
**Pull request by slovern**
Release Notes -  New Features - Telco RAN 4.15

Version(s): 4.15

Issue: https://issues.redhat.com/browse/TELCODOCS-1562

Link to docs preview: 

- Hub-side templating for PolicyGenTemplate CRs
https://71585--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-hub-side-templating-policygentemplate
- Node Tuning Operator
https://71585--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-nto-latency-testing

QE review:
- [x] QE has approved this change.

**QE Approvers**:
- Hub-side templating for PolicyGenTemplate CRs - [josclark42](https://github.com/josclark42)
- NTO - [shajmakh](https://github.com/shajmakh)

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -652,6 +652,12 @@ The Network Observability Operator releases updates independently from the {prod
 [id="ocp-4-15-scalability-and-performance"]
 === Scalability and performance
 
+[id="ocp-4-15-hub-side-templating-policygentemplate"]
+==== Hub-side templating for PolicyGenTemplate CRs
+You can manage the configuration of fleets of clusters by using hub templates to populate the group and site values in the generated policies that get applied to managed clusters.
+By using hub templates in group and site `PolicyGenTemplate` (PGT) CRs you can significantly reduce the number of policies on the hub cluster
```
Comment (irinamihai) [inline release_notes/ocp-4-15-release-notes.adoc:658]: Missing period after "hub cluster".

**Comment by irinamihai** [comment]
For whatever reason there are double quotes at the end of the paragraph [title](https://71585--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes#ocp-4-15-hub-side-templating-policygentemplate).

**Comment by josclark42** [comment]
LGTM

**Comment by shajmakh** [comment]
NTO section LGTM. Thanks @slovern.

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Pull request merged**